pkg/optimizer/datapath_csp.go

```diff
@@ -152,8 +152,9 @@ func (dpc *DataPathCSP) addInterface(intfc *taxonomy.Interface) {
 
 // This is the main method for building a FlatZinc CSP out of the data-path parameters and constraints.
 // Returns a file name where the model was dumped
-// NOTE: Minimal index of FlatZinc arrays is always 1. Hence, we use 1-based modeling all over the place to avoid confusion
-//       The only exception is with interfaces (0 means nil)
+//
```

Comment:
Remove the empty line

Reply:
I think it is better to separate the `Note:`